dev/io.openliberty.jcache.internal_fat.hazelcast/fat/src/io/openliberty/jcache/internal/fat/JCacheDynamicUpdateTest.java

```diff
@@ -0,0 +1,120 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.jcache.internal.fat;
+
+import static org.junit.Assert.assertTrue;
+
+import java.util.UUID;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.log.Log;
+
+import componenttest.annotation.CheckForLeakedPasswords;
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.custom.junit.runner.Mode;
+import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.topology.impl.LibertyServer;
+import io.openliberty.jcache.internal.fat.plugins.TestPluginHelper;
+
+/**
+ * Contains distributed JCache test dynamic server.xml changes
+ */
+@RunWith(FATRunner.class)
+@Mode(TestMode.FULL)
+public class JCacheDynamicUpdateTest extends BaseTestCase {
+    private static final Class<?> c = JCacheDynamicUpdateTest.class;
+    @Server("io.openliberty.jcache.internal.fat.dynamicupdate.1")
+    public static LibertyServer server1;
+
+    @Before
+    public void before() throws Exception {
+        String groupName = UUID.randomUUID().toString();
+
+        /*
+         * Start server 1.
+         */
+        startServer1(server1, groupName, 25000, 10);
```

Comment:
We can default size and TTL.
```suggestion
        startServer1(server1, groupName, null, null);
```